x/posts/keeper/keeper.go

```diff
@@ -16,12 +16,16 @@ type Keeper struct {
 	paramsSubspace paramstypes.Subspace
 	hooks          types.PostsHooks
 
+	ak types.ProfilesKeeper
```

Comment:
I've named the profiles keeper `ak` as inside our codebase the `pk` is used to refer to the posts keeper and I didn't want to rename all the references. Since profiles are accounts, I think `ak` (short term for account keeper) is fine as well